### DISCUSSION TITLE
articles/how-to-make-your-own-nextjs-starter-template

### DISCUSSION THREAD

**Discussion by giscus** [yehezkielgunawan/yehezgun-new#15] [Article Post]
# articles/how-to-make-your-own-nextjs-starter-template

Yehezkiel Gunawan's Article Post

https://yehezgun.com/articles/how-to-make-your-own-nextjs-starter-template

**Comment by yehezkielgunawan** [comment]
### 👋 Hi there! Hope you're all good.
You can discuss about this topic here or share this readings if you like it 😁
